pkg/v1/providers/ytt/02_addons/management-packages/add_featuregates_package.yaml

```diff
@@ -0,0 +1,195 @@
+#@ load("@ytt:data", "data")
+#@ load("@ytt:yaml", "yaml")
+
+#@ def featuregates_package_install_definition():
+---
+apiVersion: v1
+kind: ServiceAccount
+metadata:
+  name: tanzu-featuregates-package-sa
+  namespace: tkg-system
+---
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: tanzu-featuregates-package-cluster-role
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    verbs:
+      - create
+      - update
+      - delete
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - ""
+    resources:
+      - serviceaccounts
+      - services
+    verbs:
+      - get
+      - create
+      - update
+      - watch
+      - list
+  - apiGroups:
+      - apps
+    resources:
+      - deployments
+    verbs:
+      - create
+      - update
+      - delete
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - apiextensions.k8s.io
+    resources:
+      - customresourcedefinitions
+    verbs:
+      - create
+      - update
+      - delete
+      - get
+      - list
+      - watch
+  - apiGroups:
+      - cert-manager.io
```

Comment:
Is cert-manager being used in feature gate package?

Reply:
yes, we need certificates for the featuregates webhook